composer.json

```diff
@@ -11,7 +11,7 @@
     ],
     "require": {
         "php": ">=8.1",
-        "helmich/typo3-typoscript-parser": "^2.3.1",
+        "helmich/typo3-typoscript-parser": "dev-master#0ccb3a6",
         "symfony/var-exporter": "^5.3",
```

Comment:
This is just temporary requirement for the PR with fixed Symfony version: https://github.com/martin-helmich/typo3-typoscript-parser/pull/74

In stable package this would be problem, but in Rector we build the final scoped version from dev packages as well, so we're ok :+1: 

As soon the fixed PR is released, we can switch back to stable tag